src/components/Zap/ZapAction/ZapAction.tsx

```diff
@@ -0,0 +1,265 @@
+'use client';
+import { useTranslation } from 'react-i18next';
+
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+import InfoIcon from '@mui/icons-material/Info';
+import LanguageIcon from '@mui/icons-material/Language';
+import TelegramIcon from '@mui/icons-material/Telegram';
+import XIcon from '@mui/icons-material/X';
+import {
+  alpha,
+  Box,
+  Container,
+  Skeleton,
+  Typography,
+  useTheme,
+} from '@mui/material';
+import Image from 'next/image';
+import { useState } from 'react';
+import { AccordionFAQ } from 'src/components/AccordionFAQ';
+import type { TabProps } from 'src/components/Tabs';
+import { Tabs } from 'src/components/Tabs';
+import { getSiteUrl } from 'src/const/urls';
+import { useMenuStore } from 'src/stores/menu';
+import type { Quest } from 'src/types/loyaltyPass';
+import type { QuestDetails } from 'src/types/questDetails';
+import { getStrapiBaseUrl } from 'src/utils/strapi/strapiHelper';
+import { BackButton } from '../BackButton/BackButton';
+import {
+  ZapActionProtocolCard,
+  ZapActionProtocolDisclaimer,
+  ZapActionProtocolIntro,
+  ZapActionProtocolShare,
+  ZapActionProtocolShareLink,
+  ZapProtocolActionBox,
+  ZapProtocolActionInfoBox,
+} from './ZapAction.style';
+import { ZapActionFaqAccordionHeader } from './ZapActionFaqAccordionHeader';
+import { ZapWidget } from './Zapwidget';
+
+interface ZapActionProps {
+  market?: Quest;
+  detailInformation?: QuestDetails;
+}
+
+export const ZapAction = ({ market, detailInformation }: ZapActionProps) => {
+  const [tab, setTab] = useState(0);
+  const { setSnackbarState } = useMenuStore((state) => state);
+  const { t } = useTranslation();
+  const baseUrl = getStrapiBaseUrl();
+  const theme = useTheme();
+
+  const containerStyles = {
```

Comment:
Why no styled co,ponent?

Reply:
You have brought this up before, but I really do not know how to apply that. Maybe we can have a chat on what you would image the solution to look like: 

The <Tabs> functional-component currently passes those style-props accordingly as an sx to the actual styled-components. Although, I´d love to not pass on such a CSSObject, I don´t know how. 

Maybe you have an example how to solve?

Reply:
@tcheee Discussing it in the next dedicated call.